GOVERNANCE.md

```diff
@@ -0,0 +1,124 @@
+time, a [Board of Directors](#Board of Directors) is established with it's
+own set of responsibilities, to help manage the Diamond Bluff project.
+
+## Membership
+
+The Diamond Bluff project is composed of Platinum, Silver, and Participating
+Academic members. All Platinum and Silver members must be current corporate
+members of the hosting foundation (e.g. Linux Foundation or Open Infrastructure
+Foundation) at any level to participate in the Diamond Bluff project.
+
+Platinum, Silver and Participating Academic Members shall be entitled to:
+
+* Participate in the Diamond Bluff Project meetings, initiatives, events and
+  any other activities.
+* Identify their company or academic institution as a member or participant in
+  the Diamond Bluff project.
+* Platinum members shall be entitled to appoint a representative to the
+  Board of Directors of the Diamond Bluff project, as well as any future
+  committees established by the Board of Directors.
+
+## Structure
+
+At a high level, the Diamond Bluff project is governed by two main bodies which
+work together to achieve the project's goals:
+
+* A Board of Directors
+* A Technical Steering Committee
+
+### Board of Directors
+
+The Diamond Bluff project Board of Directors shall be composed as follows:
+
+* One representative from each Platinum member.
+* One Silver member representative elected annually by the Silver members for
+  every 5 Silver members, with at least one representative, and no more than 3.
+
+The Board of Directors shall have the following responsibilities:
+
+* Approve a budget directing the use of funds raised from all sources of
+  revenue.
+* Elect a Chair of the Diamond Bluff project to preside over meetings,
+  authorize expenditures approved by the budget, and manage any day-to-day
+  operations, and vote on decisions or matters before the Board of Directors.
+
+The Board of Directors shall::
+
+* Demonstrate plans and the means to coordinate with the open source project’s
+  developer community, including on topics such as branding, logos, and other
+  collateral that will represent the community.
+* Engage in a professional manner consistent with maintaining a cohesive
+  community, while also maintaining the goodwill and esteem of the project's
+  foundation, in the open source software community.
+  community.
+* Respect the rights of all trademark owners, including any branding and usage
+  guidelines.
+* Engage the project's foundation for all press and analyst relations activities.
+* Upon request, provide information regarding project participation, including
+  information regarding attendance at project-sponsored events, to the
+  project's foundation.
+* Engage the project's foundation for any Diamond Bluff project websites, and
+  operate under such rules and procedures as may from time to time be approved
+  by the Diamond Bluff Board of Directors and confirmed by the project's
+  foundation.
+
+### Technical Steering Committee
```

Comment:
Alternate version:

### Technical Steering Committee

Members of the Diamond Bluff Technical Steering Committee (TSC) decide technical 
matters that affect the entire project, administer joint project-wide 
technical resources, and act as a final arbiter of technical matters that are 
not decided inside individual subprojects.

#### TSC Membership

All TSC members must be currently contributing engineers, documentation writers,
test platform adminstrators, specification editors, or other direct technical 
contributors to one or more subprojects.

During the first year of of the project, The TSC shall be composed as follows:

* One appointed representative from each Platinum member
* One maintainer from each Diamond Bluff sub-project, selected by the Board

Starting one year after the launch of the Diamond Bluff governance, the TSC 
composition will become:

* Two maintainers from each Diamond Bluff sub-project
* Two technical contributors appointed by the Board

Membership in the TSC will gradually transition to a model of elected subproject
technical leadership, starting in the second year of the project. Details for 
this transition will be decided by the Board.

Should any TSC member cease regular technical contributions, their membership
will terminate at the next regular annual selection date. If a TSC member
violates the Code of Conduct, they may be removed from the TSC by the Code
of Conduct Committee.

#### TSC Responsibilities

The responsibilities of the Diamond Bluff Technical Steering Committee are
as follows:

* Coordinating the technical direction of the Diamond Bluff Project
* Nominating and reviewing new sub-projects for creation under the Diamond Bluff 
  organization after final approval by the Board.
* Confirming selected maintainers in the sub-projects, and ensuring they have the
  technical support necessary to be successful in their role as a maintainer.
* Communicating with external and industry organizations concering Diamond
  Bluff technical matters.
* Appointing represenatives to work with other open source or open standards
  communities.
* Voting on technical matters relating to any common code base or specifications
  affecting all sub-projects.
* Adminstering common technical resources, such as repository hosting and the 
  testing infrastructure.
* Acting as "court of final appeal" for technical matters which are not resolved
  through normal decision mechanisms within the individual subprojects.
* Coordinating response to security issues and ensuring that all subprojects
  are reasonably secure.
  
For any of the above responsibilities, the TSC may delegate their powers to one
or more parties as appropriate and necessary, under terms set by the TSC.

#### TSC Meetings

The TSC shall hold one or more public meetings each month.  All contributors
to, or sponsors of, Diamond Bluff shall be entitled to attend these meetings. 
Public meetings will have a public agenda to which members of the community may
reasonably add items.

The TSC may also have private meetings from time to time, particularly when
discussing security response or settling escalated disputes. All TSC members
will be included in private meetings.

Reply:
This version does several things:  

1. Ensures that the TSC is made up of people actually working on the subprojects
2. Adds the expectation that the TSC will transition to a model of technical promotion from below instead of appointment by the Board, but leaves that exact model flexible.
3. Adds a more complete set of TSC responsibilities.

Reply:
I like this version, thanks for the help!